Tests/Common/TestUtils.cs

```diff
@@ -133,7 +133,7 @@ public static Twin CreateTwin(Dictionary<string, object> desired = null, Diction
                 { TwinProperty.DevAddr, simulatedDevice.DevAddr },
                 { TwinProperty.AppSKey, simulatedDevice.AppSKey },
                 { TwinProperty.NwkSKey, simulatedDevice.NwkSKey },
-                { TwinProperty.DevNonce, simulatedDevice.DevNonce },
+                { TwinProperty.DevNonce, simulatedDevice.DevNonce.ToString("N", null) },
```

Comment:
I would question why this needs to be in hexadecimal.

Reply:
Changed in 9f3eb36e738fbaa1a3f376ab154540aa659eb33c